indra_db/managers/content_manager.py

```diff
@@ -835,11 +835,9 @@ def dump_annotations(self, db):
                             annotation['major_topic'], is_concept)
 
                 # Handle the qualifier
-                qual = annotation['qualifier']
```

Comment:
`qualifier` is still a valid key and its value is a single qualifier for compatibility with this bit of code so this change isn't strictly necessary

Reply:
Not strictly, but in the INDRA repo it describes the inclusion of "qualifier" as being strictly for the sake of the indra database, specifically this bit of code. It makes sense to me to untether INDRA from that obligation, especially because ultimately we probably want to have multiple qualifiers saved in the database.